.travis.yml

```diff
@@ -20,7 +20,7 @@ install:
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy pytest
   - source activate test-environment
   - easy_install --upgrade six
-  - pip install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.10.0-cp27-none-linux_x86_64.whl
+  - pip install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0-cp27-none-linux_x86_64.whl
```

Comment:
Oh, I  was going to do an 10->11 update then an 11->12 update in this one pull request, but when the 10->11 went funny, I wanted to figure out why before moving on.